nuxeo-drive-client/scripts/ndrive.py

```diff
@@ -1,6 +1,14 @@
 #!/usr/bin/env python
 """Commandline interface for the Nuxeo Drive filesystem synchronizer"""
+try:
+    from nxdrive.commandline import main
+except ImportError:
```

Comment:
In the tests that are still using the direct call to ndrive.py. This is in my list to refactor them.